addOns/pscan/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Updated Automation Framework example yaml files and help content for passiveScan-config jobs to be more consistent.
```

Comment:
- "example yaml files" -> "template plans"
- This should be done for both jobs.